src/twisted/test/test_defer.py

```diff
@@ -1678,6 +1679,26 @@ def test_fromCoroutineRequiresCoroutine(self) -> None:
         for thing in thingsThatAreNotCoroutines:
             self.assertRaises(defer.NotACoroutineError, Deferred.fromCoroutine, thing)
 
+    @ensuringDeferred
+    async def test_fromCoroutineWithEnsureFuture(self) -> None:
+        """
+        L{Deferred.fromCoroutine} should properly process pending futures
```

Comment:
What does "properly" means ?  

I don't know why this has no coverage.
Is this test executed?

Reply:
> What does "properly" means ?

It means we should back control the event loop before we can call `send` again. Not sure how to describe it. 
And yes, this test is executed, at least locally, and without these changes is failed.

Reply:
> It means we should back control the event loop before we can call send again.

Then the docstring should say those words, rather than "properly" :-) 

See https://jml.io/test-docstrings/

Reply:
@glyph thanks! Good page.